docs/subscriptions/update.md

```diff
@@ -0,0 +1,171 @@
+# Update a subscription
+
+:::{include} /fragments/core-admonition.md
```

Comment:
Should we make a note (or add a different include) explaining that the endpoint is expected to be supported, but not all calls (and then denote requirements at the level of calls?